.buildkite/scripts/steps/code_generation/packages_codegen.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+echo --- OpenAPI Packages Code Generation
+
+(cd packages/kbn-openapi-common && yarn openapi:generate)
+check_for_changed_files "yarn openapi:generate" true
+
+(cd packages/kbn-securitysolution-lists-common && yarn openapi:generate)
+check_for_changed_files "yarn openapi:generate" true
```

Comment:
Nit: This might belong more to the `.buildkite/scripts/steps/code_generation/security_solution_codegen.sh`. Not a strong opinion, but, for example, `elastic_assistant_codegen.sh` has it's own script and it generates code for a package.

Reply:
Good point. I moved it to `security_solution_codegen.sh` script.